csharp-selenium-webdriver-sample/azure-pipelines.yml

```diff
@@ -29,52 +29,78 @@
-  - task: UseDotNet@2
-    displayName: install .NET 6.0.415
-    inputs:
-      version: '6.0.415'
+    stages:
+    - stage: Stage
+      jobs:
+      - ${{ each item in parameters.strategy }}:
+        - job: Job_${{ item.name }}
+          pool:
+            name: $(a11yInsightsPool)
+            image: ${{ item.imageName }}
+            os: ${{ item.os }}
+          variables:
+            # This will be sent to the build agent as an environment variable which our test code will be able to see.
+            # We'll use this environment variable in WebDriverFactory.cs to determine which browser the tests will use.
+            SELENIUM_BROWSER: ${{ item.browser }}
+            # This skips unnecessary steps during dotnet restore and saves ~30-60s
+            DOTNET_SKIP_FIRST_TIME_EXPERIENCE: '1'
 
-  - task: DotNetCoreCLI@2
-    displayName: dotnet restore
-    inputs:
-      command: restore
-      projects: $(Build.SourcesDirectory)/csharp-selenium-webdriver-sample/**/*.csproj
+          steps:
+            # This task makes sure .NET Core SDK is installed on the build agent.
+            #
+            # It is a good idea to test against a specific, known version of .NET SDK; this makes it easier for other users
+            # to reproduce your build results.
+          - task: UseDotNet@2
+            displayName: install .NET 6.0.415
+            inputs:
+              version: '6.0.415'
 
-  # Using DotNetCoreCLI rather than a "script" step that runs "dotnet test" makes it easier to publish the test results
+          - task: DotNetCoreCLI@2
+            displayName: dotnet restore
+            inputs:
+              command: restore
+              projects: $(Build.SourcesDirectory)/csharp-selenium-webdriver-sample/**/*.csproj
+
+              # Using DotNetCoreCLI rather than a "script" step that runs "dotnet test" makes it easier to publish the test results
   # to see along with the rest of the build details in the Azure Pipelines web UI.
-  - task: DotNetCoreCLI@2
-    displayName: dotnet test
-    inputs:
-      command: test
-      projects: $(Build.SourcesDirectory)/csharp-selenium-webdriver-sample/**/*.csproj
-      publishTestResults: true
-      testRunTitle: "dotnet test ($(imageName) $(browser))"
-      # We use this EXTRA_DOTNET_TEST_ARGUMENTS variable in the sample's CI/PR builds so the "success" examples can --filter
+          - task: DotNetCoreCLI@2
+            displayName: dotnet test
+            inputs:
+              command: test
+              projects: $(Build.SourcesDirectory)/csharp-selenium-webdriver-sample/**/*.csproj
+              publishTestResults: true
+              testRunTitle: "dotnet test (${{ item.imageName }} ${{ item.browser }})"
+               # We use this EXTRA_DOTNET_TEST_ARGUMENTS variable in the sample's CI/PR builds so the "success" examples can --filter
       # out the tests that intentionally fail for example purposes. You probably don't need this in a real project.
```

Comment:
Lines 103 & 104 should have the same indent

Reply:
Changes done.